v8env/rollup.config.js

```diff
@@ -67,6 +68,28 @@ function strings(
   };
 }
 
+function runtimeInfo(path) {
+  const filter = createFilter([path]);
+
+  return {
+    name: "runtimeInfo",
+    transform: (code, id) => {
+      if (filter(id)) {
+        const build = execSync('./scripts/build-number.sh', {
+          cwd: '..'
+        }).toString().trim();
+
+        const magicString = new MagicString(`export const runtime = { build: "${build}" };`);
```

Comment:
This is an interesting package. I was gonna suggest https://github.com/rollup/rollup-plugin-replace, but it appears to be using magic-string itself!